script/run-mockapi.sh

```diff
@@ -7,7 +7,8 @@ trap 'if [ $(jobs -p) ] ; then kill $(jobs -p); fi' EXIT
 if [ "$(nc -z localhost 3000; echo $?)" -ne 0 ]; then
     echo "Starting mockapi.js..."
     node src/platform/testing/e2e/mockapi.js &
-    sleep 5;
+    # waits for process to stop
+    wait;
```

Comment:
`sleep` was causing the mock-api to stop before the tests finish. `wait` should just wait for the process to end to stop the server.